spec/lib/tasks/evm_export_import_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rake'
+
+describe 'evm_export_import' do
```

Comment:
This file really isn't needed. Testing the prerequisites of a rake task is a bit extreme.